PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -20,10 +20,16 @@
   <change-notes>
     <![CDATA[
     <html>
-    <h3>3.21.1</h3>
+    <h3>3.22.0</h3>
+    <h4>Added</h4>
+    <ul>
+    </ul>
+    <h4>Changed</h4>
+    <ul>
+        <li>Improve app service data loading performance</li>
```

Comment:
How about the note for "added the vscode format telemetry", which might help us to trace the version and telemetry data relationship?

Reply:
I'm not sure whether we should add events about telemetry in the release note. It do help us but users may not benefit from that.